src/high-level/cursor.lisp

```diff
@@ -57,3 +57,20 @@
     (move-cursor window
                  (max 0 (- x amount))
                  y)))
+
+
+(defun cursor-visibility (visibility)
+  "Set the cursor to VISIBILITY. VISIBILITY can be 0, 1, 2 or the symbols
+'invisible, 'normal, 'very-visible which correspond to the numbers 
+respectively. Returns the previous state if the change is succesful, else
+it returns nil or returns an error if the mode is not supported."
+  (check-status
+   (let ((vis-change (cond
+					   ((or (eq visibility 0) (eq visibility 'invisible)) 0)
```

Comment:
I'm not sure how this should be tackled. The man page uses the terms invisible, normal, and very visible, so I feel that they should be added in some regard for ease of use. I've left it out of exports until something is figured out.